ocaml/testsuite/tests/mixed-blocks/test_mixed_blocks.ml

```diff
@@ -12,20 +13,7 @@
 (*****************************************)
 (* Prelude: Functions on unboxed numbers *)
 
-module Float_u = struct
-  include Stdlib__Float_u
-
-  let ( + ) = add
-  let ( - ) = sub
-  let ( * ) = mul
-  let ( / ) = div
-  let ( ** ) = pow
-  let ( > ) x y = (compare x y) > 0
-end
-
-module Int32_u = Stdlib__Int32_u
-module Int64_u = Stdlib__Int64_u
-module Nativeint_u = Stdlib__Nativeint_u
+open Stable
```

Comment:
I would prefer to explicitly have `module Float_u = Stable.Float_u` etc. That way you get more of a suggestion just from scanning the prelude which kinds of mixed blocks are tested in this file.